packages/appstore/src/components/cfds-listing-logged-out/cfds-listing-logged-out.tsx

```diff
@@ -22,11 +22,13 @@ const CFDsListingLoggedOut = observer(() => {
 
     return (
         <ListingContainer title={<CFDsTitle />} description={<CFDsDescription />}>
-            <div className='cfds-listing-logged-out__cfd-full-row'>
-                <Text line_height='m' weight='bold' color='prominent'>
-                    {localize('Deriv MT5')}
-                </Text>
-            </div>
+            {combined_cfd_mt5_accounts.length > 0 && (
+                <div className='cfds-listing-logged-out__cfd-full-row'>
+                    <Text line_height='m' weight='bold' color='prominent'>
+                        {localize('Deriv MT5')}
```

Comment:
pls check if there's a config we can use to avoid this hardcoding, like `getCFDPlatformLabel` from @shared

Reply:
I just moved it inside condition